marlowe-runtime/doc/ReadMe.md

```diff
@@ -1,100 +1,141 @@
 
-Knowledge of Haskell or Plutus is not required for developers to communicate with Runtime. Communication with Runtime over REST and/or WebSocket APIs is language-agnostic, so developers can use a language of their choice (e.g., Python, JavaScript, or Rust). However, the Runtime backend can interact directly with Haskell-based clients.
+Communication with Runtime can be done over many language-agnostic APIs (REST,WebSocket and AWS-λ) allowing developers to use the language of their choice (e.g., Python, JavaScript, Rust, ...). Knowledge of Haskell or Plutus is not required for developers. However, the Runtime backend can interact directly with Haskell-based clients.
 
+## Architecture
 
+![Marlowe Runtime ecosystem](diagrams/ecosystem.png)
 
-## Architecture
+Marlowe runtime consists of a:
+- Chain-indexing service (`chainseekd`), 
+- History-tracking service for Marlowe contracts (`marlowe-history`), 
+- Discovery service for Marlowe contracts (`marlowe-discovery`), 
+- Transaction-creation service for Marlowe contracts (`marlowe-tx`). 
 
-The backend for Marlowe runtime consists of a chain-indexing service (`chainseekd`), a history-tracking service for Marlowe contracts (`marlowe-history`), a discovery service for Marlowe contracts (`marlowe-discovery`), and a transaction-creation service for Marlowe contracts (`marlowe-tx`). These backend services operate in concert and rely upon [cardano-node](https://github.com/input-output-hk/cardano-node/blob/master/README.rst) for blockchain connectivity and upon [PostgreSQL](https://www.postgresql.org/) for persistent storage. Access to the backend services is provided via a command-line client (`marlowe`), an AWS Lambda function (`marlowe-lambda`), or a REST/WebSocket server (`web-server`) that uses JSON payloads. Web applications can integrate with a [CIP-30 light wallet](https://cips.cardano.org/cips/cip30/) for transaction signing, whereas enterprise applications can integrate with [cardano-wallet](https://github.com/input-output-hk/cardano-wallet/blob/master/README.md), [cardano-cli](https://github.com/input-output-hk/cardano-node/blob/master/cardano-cli/README.md), or [cardano-hw-cli](https://github.com/vacuumlabs/cardano-hw-cli/blob/develop/README.md) for signing transactions.
+Operating in concert and relying upon  : 
+- [cardano-node](https://github.com/input-output-hk/cardano-node/blob/master/README.rst) for blockchain connectivity,
+- [PostgreSQL](https://www.postgresql.org/) for persistent storage. 
 
-![Marlowe Runtime ecosystem](diagrams/ecosystem.png)
+Marlowe runtime functionalities are accessible via: 
+- Command-line client (`marlowe`), 
+- AWS-λ function (`marlowe-lambda`), 
+- REST/WebSocket server (`web-server`) that uses JSON payloads. 
 
+For transaction signing :
```

Comment:
It might be worth pointing out that the runtime is completely wallet-agnostic, alongside examples of wallets that can be used alongside it.

Reply:
replaced it with : 
```
For transaction signing, Marlowe Runtime is completely wallet-agnostic and therefore all the different implementations available can be used :
 - Web applications can integrate with a [CIP-30 light wallet](https://cips.cardano.org/cips/cip30/)
 - Enterprise applications can integrate with : 
	- [cardano-wallet](https://github.com/input-output-hk/cardano-wallet/blob/master/README.md), 
	- [cardano-cli](https://github.com/input-output-hk/cardano-node/blob/master/cardano-cli/README.md), 
	- [cardano-hw-cli](https://github.com/vacuumlabs/cardano-hw-cli/blob/develop/README.md).
```